app/models/symphony/patron.rb

```diff
@@ -223,6 +223,12 @@ def can_pay_fines?
       true
     end
 
+    def payments
```

Comment:
If we are merging new code related to the Symphony client, should there be a new test?

Reply:
this (and `symphony_legacy_client`) came over from the `SymphonyClient`, so it's more of a move than new code imo. I just needed it to be here instead of on the client, so that `patron.payments` worked for both Symphony and FOLIO